docs/maps/create-custom-maps-for-highmaps.md

```diff
@@ -49,13 +49,14 @@ Highmaps can be used for more than geographic data, and more than the featured m
 
 15. Go to [jsfiddle.net/highcharts/TUy7x/](https://jsfiddle.net/highcharts/TUy7x/), an empty boilerplate for Highmaps. Add a series configuration object and paste your data. It should now look like [jsfiddle.net/highcharts/TUy7x/1/](https://jsfiddle.net/highcharts/TUy7x/1/).
 
-    
-    $(function () {
-        // Initiate the chart
-        $('#container').highcharts('Map', {
-            series: /* paste your content here */
-        });
+```js
+$(function () {
+    // Initiate the chart
+    $('#container').highcharts('Map', {
```

Comment:
Do we need jQuery solution?

Reply:
Probably not, note my PR does not remove jQuery from docs, it's rather focused on code formatting.

Quick search shows that there's ~10 other places where jQuery is used in docs.

Reply:
For the records, they should all be removed.

Reply:
👍